web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -560,6 +560,50 @@ describe("<BusinessFormationPaginator />", () => {
         expect(screen.queryByText(Config.formation.errorBanner.errorOnStep)).not.toBeInTheDocument();
         expect(page.getStepStateInStepper(LookupStepIndexByName("Billing"))).toEqual("COMPLETE-ACTIVE");
       });
+
+      it("shows the Trustees label when members' field has an error and legalType is nonprofit", async () => {
+        const nonprofit = generateBusiness({
+          formationData: generateFormationData({
+            formationFormData: generateFormationFormData({
+              members: undefined,
+            }),
+          }),
+          profileData: generateProfileData({ legalStructureId: "nonprofit" }),
+        });
+        const page = preparePage({ business: nonprofit, displayContent });
+
+        await page.stepperClickToReviewStep();
+        await page.submitReviewStep();
+
+        await page.stepperClickToContactsStep();
+        expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.errorBanner.errorOnStep);
+        expect(screen.queryByRole("alert")).not.toHaveTextContent(Config.formation.fields.members.label);
+        expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.fields.trustees.label);
+      });
+
+      const legalTypes = ["c-corporation", "s-corporation"];
```

Comment:
I think we can use the `corpLegalStructures` here as well

Reply:
Done